src/app/public/index.ts

```diff
@@ -1 +1,6 @@
+// Export the token args interface from auth client.
+export {
```

Comment:
IIRC, we wanted to limit exposing `BBAuth`.  I realize we're using it internally, but I believe that was the trepidation in adding the quick fix like you've done here.

Reply:
Ah, you are correct; I misunderstood the requirement. I'll push a change in a minute.

Reply:
Done!